app/src/main/java/com/example/fashi_shop/adapter/AdapterCart.java

```diff
@@ -1,76 +1,88 @@
 
 import java.text.DecimalFormat;
 import java.text.NumberFormat;
 import java.util.List;
 
-public class AdapterGioHang extends RecyclerView.Adapter<AdapterGioHang.ViewHolderGioHang> {
+public class AdapterCart extends RecyclerView.Adapter<AdapterCart.ViewHolderGioHang> {
 
     Context context;
     List<Product> products;
     int layout;
 
-    public AdapterGioHang(Context context, List<Product> products, int layout){
+    public AdapterCart(Context context, List<Product> products, int layout){
         this.context = context;
         this.products = products;
         this.layout = layout;
     }
 
     public class ViewHolderGioHang extends RecyclerView.ViewHolder {
         ImageView imageProduct, imageDelete;
-        TextView name, price;
+        ImageButton quantityUp, quantityDown;
+        TextView name, price, quantity;
 
         public ViewHolderGioHang(View itemView) {
             super(itemView);
             imageProduct = itemView.findViewById(R.id.item_image_cart);
             imageDelete =  itemView.findViewById(R.id.imDeleteItem);
+            quantityUp = itemView.findViewById(R.id.ibQuantityUp);
+            quantityDown = itemView.findViewById(R.id.ibQuantityDown);
             name = itemView.findViewById(R.id.item_product_name_cart);
             price = itemView.findViewById(R.id.item_product_price_cart);
+            quantity = itemView.findViewById(R.id.txtQuantity);
         }
     }
 
 
     @Override
-    public AdapterGioHang.ViewHolderGioHang onCreateViewHolder(ViewGroup parent, int viewType) {
+    public AdapterCart.ViewHolderGioHang onCreateViewHolder(ViewGroup parent, int viewType) {
         Context context = parent.getContext();
         LayoutInflater layoutInflater = LayoutInflater.from(context);
         View view = layoutInflater.inflate(layout, parent, false);
-        ViewHolderGioHang viewHolderGioHang = new ViewHolderGioHang(view);
-        return viewHolderGioHang;
+        ViewHolderGioHang viewHolderCart = new ViewHolderGioHang(view);
+        return viewHolderCart;
     }
 
     @Override
-    public void onBindViewHolder(AdapterGioHang.ViewHolderGioHang holder, int position) {
+    public void onBindViewHolder(AdapterCart.ViewHolderGioHang holder, int position) {
         Product product = products.get(position);
 
         holder.name.setText(product.getName());
 
         NumberFormat formatter = new DecimalFormat("#,###");
         holder.price.setText(formatter.format(product.getPrice()) + "₫");
 
+
         Bitmap bitmapImageCart = BitmapFactory.decodeByteArray(product.getImage_gio_hang(), 0, product.getImage_gio_hang().length);
         holder.imageProduct.setImageBitmap(bitmapImageCart);
 
         holder.imageDelete.setTag(product.getId());
         holder.imageDelete.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-
+                Cart cart = new Cart();
+                cart.ConnectSQL(context);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.